New integration

DufourVictor opened the pull request:
I'm bored during this confinement so I propose a new graphic identity to **Elephpant.me**

I couldn't find where the "verify.blade.php" template was used so I didn't style it. If used tell me how I can access it and I will style this.

Hope you like this MR and I'm happy to be able to contribute as much as I can to this project.
If you have any questions don't hesitate.

DufourVictor committed:
New integration

jgrossi commented:
hey man! thanks so much for the contribution. gonna pull to my local and check the changes 🙏

DufourVictor commented:
Hope that you'll enjoy it :smiley:

jgrossi commented:
hey man! first of all, thank you! amazing work!!! we have some comments (some of them we need to fix I think):
- the table rows (gray background) is not 100% readable, so maybe it's a good idea to have a lighter bg to improve readability. can you change that please?
- the logo @IgorSantos17 suggested to add the ".me" to the end, so it would be "ElePHPant.me";
- when we change (+ or - buttons) the number of elephpants in the herd we use the markup to update the total (in the stimulus js controller I think). it seems after changing the styles that's broken. that's something for sure we can fix, np

I'm gonna try to push a commit to your branch with the stimulus js change (for the total in the herd). sounds good?

and one more time! thank you!

DufourVictor committed:
Integration fixes

DufourVictor commented:
Hey! No problem, I really like this project and I'm very happy to contribute. Thanks to you for this project.

For your comments, I've fixed everything in a second commit. 
Indeed I didn't notice for the total of elephpants in the herd. It's now fixed.

Tell me if I forgot something or if you want other changes.

DufourVictor commented:
Some news? 🙂
